tutorials/07_sensitivity_analysis.ipynb

```diff
@@ -104,9 +104,10 @@
     "    )\n",
     "\n",
     "\n",
-    "posterior = infer(simulator, prior, num_simulations=2000, method=\"SNPE\").set_default_x(\n",
-    "    torch.zeros(2)\n",
-    ")"
+    "theta, x = simulate_for_sbi(simulator, prior, 2000)\n",
```

Comment:
I agree to removing `infer` 

however, I would prefer not using `simulate_for_sbi` here. Just sample from prior and can simulator. 
I think we should promote `simulate_for_sbi` mainly for simulating in parallel.